data/remnant missions.txt

```diff
@@ -1506,3 +1510,150 @@ mission "Remnant: Broken Jump Drive job"
 		dialog "The research team quickly offloads the broken jump drive and pays you <payment> before hurrying back to their lab."
 
 
+
+mission "Remnant: Face to Maw 1"
+	name "Talk to Plume about the Void Sprites"
+	description "Plume would be very interested in a report on what happened with the Void Sprites"
+	landing
+	to offer
+		has "Remnant: Return the Samples 2: done"
+	source
+		system "Nenia"
+	destination "Aventine"
+	on offer
+		conversation
+			`As the swirls of gas close over the <ship> you wonder idly what brought you back here. This time, there is no apologetic researcher asking you to risk your life for research, or to return eggs from where they were stolen. The last glimpse of the Archon drifting overhead as you descend below the cloud deck provides no answers; For now, it is nothing more than a silent, brooding presence in the skies of Nenia. The silent depths of an alien world fold over you, gradually erasing the rest of the galaxy from sight.`
+			`	As the <ship> levels off at the designated altitude, you strap yourself into the pressure suit. Stepping into the airlock, you lower the shields so you can exit onto the hull. Out on the surface, you admire the misty kaleidescope of colors filtering down through the cloud layers, playing over distant shapes moving in the mists. Before long several void sprites emerge, weaving and dancing among the clouds. They swirl for a few minutes around your cargo bay doors as if waiting for something then pulse briefly with flickers of light and then drop into the clouds below. The third void sprite flutters around the <ship>, poking at things and sliding over them. You quickly think to turn on your video recorder to catch the scene. The Void Sprite notices your brief movement, and turns towards you with a lunge. As it closes on you, your attention is drawn to what you suspect is its mouth: A zig-zagging crack that peels open like a zipper to reveal an opening that resembles an industrial grinder.`
```

Comment:
Do the shields need lowered? Shields are part of the hull itself, not something projected out into space.
Not all sci-fi shields prevent travel, either, so this is claiming something about shields in ES that I'm not sure is backed up right now.

Reply:
That's true. 

1) Shields in ES as described so far provide a purely military use, although some descriptions mention having been used historically for radiation and asteroid deflection. All these uses imply a physical barrier that prevents objects and energy from passing through it. 

2) It should also be pointed out that the gascraft is currently the only ship in the game that has gaslining, so this mission can only be triggered using it. And the gascraft is specifically described as a relic, so this description is only establishing lore regarding how antique remnant shields from centuries ago worked. These may well include the first barely-functional prototypes of alternate ship systems developed from alien ruins. so it is highly likely that they do not have all the functionality of modern shields.

3) I would also argue that a core feature of a gaslined ship is a hull constructed using different techniques. It is reasonable to think that such a hull may not incorporate shields in the same way that regular ships do. 

4) This is a research ship, and energy fields of any kind may interfere with precision equipment, especially those that try to measure minute energy fields. It makes sense to me that such a ship would include mechanisms to allow them to completely drop the shields for the sake of their study.

(caveat for 2, this mission leads to a second gaslining ship, but it is fundamentally impossible for the Penguin to be used in this mission without using an all-ships plugin. So the gascraft is the only gaslining ship the game that is possible to use for the mission)

> "Centuries ago, the Remnant built these ships in order to explore the upper atmosphere of nearby gas giants. The Archons took issue with the Remnant's research methods, and the studies were unable to be completed. The remaining gascraft have been gathering dust in storage for centuries while the Remnant waits to see if the anger of the Archons will subside."

Potential solutions:
- Add description making one or more of the above points explicit so that the player doesn't assume this lore is how all ES shields function. (this is my preferred solution, as I like the idea of having shields function differently for different groups. I see no reason why every race's shields should function identically)
- Re work the descriptions so that the player isn't disabling shields, but instead at the appropriate time is activating an emergency shield extension to protect themselves. (fits with the generic shield lore about variably shaped shields)
- The player activates a plasma cutting torch on their suit and chops off the tentacle. (Possible, but the game makes a point of the character not being particularly adept at physical combat related things)

Reply:
While it doesn't fully resolve this debate, some of the conversation in the Tech Retrieval mission rewrite (#4269 ) includes some commentary on the gascraft, and mentions how said ship uses a lot of archaic technology, including 'hard-shell' shields.

> "You've flown the gascraft, you know how old that ship is: primitive energy systems, antique 'hard-shell' shields that have to be manually raised and lowered..." she trails off into momentary silence. "Be glad that you got one that had already been refurbished prior to your visit to Nenia. At least it had modern engines and an updated sensor array."